### DISCUSSION TITLE
[Suggestion] Ban inedilarion from this page.

### DISCUSSION THREAD

**Discussion by ghost** [PavelZinchenko/event-horizon-frontier-builds#228] [General]
All of his ideas are bad and do absolutely nothing to enhance the quality of life for the game. I'm tired of seeing things he suggests being implimented only to watch the game suck even more. I personally am still playing Build 1887 just to AVOID the things he has suggested and I have no intention on updating until the game is once again enjoyable.

On a side note, I am curious as to why it is his suggestions that get implimented over and before anyone else's.

**Comment by Thirteen0130** [comment]
Чел, хватит мутить воду. Пока что только ты один возмущаешься, словно истеричка. Играешь на старой версии? Ну играй дальше. Молодец. Хватит кого-то обвинять. Предлагай свои идеи и решения

**Comment by Bezbasheniy** [comment]
Предлогаю его сжечь, как Жанну Дарк!

**Comment by ghost** [comment]
> Предлогаю его сжечь, как Жанну Дарк!
(I propose to burn him like Joan of Arc!)

I agree.

**Comment by Thirteen0130** [comment]
> Чувак, хватит мутить воду. Пока ты единственный, кто возмущен, как истеричная женщина. Ты играешь на старой версии? Что ж, продолжай играть. Молодец. Перестаньте обвинять кого-либо. Предлагайте свои идеи и решения
> 
> мое решение для Павла сделать так, как он сказал 30 сборок назад, и только исправлять ошибки вместо того, чтобы продолжать добавлять дерьмо, о котором просил ОДИН ЧЕЛОВЕК.

Со стороны выглядит так, будто ты злишься, что твоя идея не была реализована, а чужая была. Вот и пытаешься сделать так, что "не достанься же ты никому". Ни себе, ни другим. Хахахахаха

**Comment by ghost** [comment]
> Чел, хватит мутить воду. Пока что только ты один возмущаешься, словно истеричка. Играешь на старой версии? Ну играй дальше. Молодец. Хватит кого-то обвинять. Предлагай свои идеи и решения
(Dude, stop muddying the waters. So far, you're the only one who is indignant, like a hysterical woman. Are you playing on the old version? Well, keep playing. Well done. Stop blaming someone. Offer your ideas and solutions)

my solution is for Pavel to do like he said 30 builds ago and only do bug fixes instead of continuing to add crap that ONE PERSON asked for.

**Comment by Bezbasheniy** [comment]
Несите факелы, автор согласен быть сожжонным🤣

**Comment by ghost** [comment]
> Со стороны выглядит так, будто ты злишься, что твоя идея не была реализована, а чужая была. Вот и пытаешься сделать так, что "не достанься же ты никому". Ни себе, ни другим. Хахахахаха
(From the outside it looks like you are angry that your idea was not implemented, but someone else’s was. So you are trying to make it so that “no one will get you.” Neither to yourself nor to others. Hahahahaha)

i havent made any suggestions. only brought up bugs. But, others have made suggestions that would actually fix some issues, and balance some things. So instead of attempting to ridicule me, how about you not waste your precious time responding to me? I am not the only one who is pissed about this. You want this game to go to hell, fine. I hope you burn with it. I also hope when this game finally hits rock bottom, you are equally blamed for it.

**Comment by Thirteen0130** [comment]
> Несите факелы, автор согласен быть сожжонным🤣

Он кажется не понял, что это мы про него. Хахахах

**Comment by ghost** [comment]
> Несите факелы, автор согласен быть сожжонным🤣
(Carry torches, the author agrees to be burned🤣)

unlike you, i used to like this game. I'm just fed up with the bullshit he suggests being added and fuckingcthe game up even more.

**Comment by Bezbasheniy** [comment]
> > Несите факелы, автор согласен сожжонным🤣
> 
> Он, кажется, не понял, что мы про это. Хахахах

Это ведь поправимо🤣 Главное что согласился 😂

**Comment by ghost** [comment]
> > Несите факелы, автор согласен быть сожжонным🤣
> 
> Он кажется не понял, что это мы про него. Хахахах
(He doesn't seem to understand that we are talking about him. Hahahaha)

exactly. dudes clueless, and not just about the game apparently.

**Comment by Thirteen0130** [comment]
> > > Несите факелы, автор согласен сожжонным🤣
> > 
> > 
> > Он, кажется, не понял, что мы про это. Хахахах
> 
> Это ведь поправимо🤣 Главное что согласился 😂

Мы его задели. Он стал переходить на личность. Ахахахах

**Comment by ghost** [comment]
speak english like you do in the discord server. im not translating anymore of your russian.

**Comment by Thirteen0130** [comment]
> > > Несите факелы, автор согласен быть сожжонным🤣
> > 
> > 
> > Он кажется не понял, что это мы про него. Хахахах
> > (He doesn't seem to understand that we are talking about him. Hahahaha)
> 
> exactly. dudes clueless, and not just about the game apparently.

Чувак, раз и навсегда скажу тебе. Ты сейчас буквально обсираешь нашу любимую игру. Мы, конечно же, будем её защищать. Даже если она и превратится в помойку, как ты говоришь, то мы будем продолжать играть в неё, потому что любим её именно за это. За возможность посмеяться и повеселиться. Это ведь игра, не нужно всё доводить до крайностей. Разработчик старается выслушать всех и сделать так, чтоб все были довольны. Но ведь всё равно будут те, кому что-то не нравится. Такова человеческая натура. Сколько людей-столько и мнений.

**Comment by ghost** [comment]
"make x+ levels harder."
"make x+ levels easier."
"add this."
"add that."

did you think any of it through before you suggested it? now jorm is dead, worm is dead, hat is useless, healers rsign supreme and its all because of YOUR GARBAGE SUGGESTIONS.

**Comment by ghost** [comment]
> > > > Несите факелы, автор согласен быть сожжонным🤣
> > > 
> > > 
> > > Он кажется не понял, что это мы про него. Хахахах
> > > (He doesn't seem to understand that we are talking about him. Hahahaha)
> > 
> > 
> > exactly. dudes clueless, and not just about the game apparently.
> 
> Чувак, раз и навсегда скажу тебе. Ты сейчас буквально обсираешь нашу любимую игру. Мы, конечно же, будем её защищать. Даже если она и превратится в помойку, как ты говоришь, то мы будем продолжать играть в неё, потому что любим её именно за это. За возможность посмеяться и повеселиться. Это ведь игра, не нужно всё доводить до крайностей. Разработчик старается выслушать всех и сделать так, чтоб все были довольны. Но ведь всё равно будут те, кому что-то не нравится. Такова человеческая натура. Сколько людей-столько и мнений.

i already told you im not translating. did you not understand that either?

**Comment by Thirteen0130** [comment]
> > > > > Несите факелы, автор согласен быть сожжонным🤣
> > > > 
> > > > 
> > > > Он кажется не понял, что это мы про него. Хахахах (Похоже, он не понимает, что мы говорим о нем. Хахахаха)
> > > 
> > > 
> > > точно. чуваки невежественны, и, по-видимому, не только в игре.
> > 
> > 
> > Чувак, раз и навсегда скажу тебе. Ты сейчас буквально обсираешь нашу любимую игру. Мы, конечно же, будем её защищать. Даже если она и превратится в помойку, как ты говоришь, то мы будем продолжать играть в неё, потому что любим её именно за это. За возможность посмеяться и повеселиться. Это ведь игра, не нужно всё доводить до крайностей. Разработчик старается выслушать всех и сделать так, чтоб все были довольны. Но ведь всё равно будут те, кому что-то не нравится. Такова человеческая натура. Сколько людей-столько и мнений.
> 
> я уже говорил вам, что я не перевожу. этого вы тоже не поняли?

Мне плевать. Русский-мой родной язык и я горжусь этим и я буду писать на нём. То, что вы не понимаете, это ваша проблема. Переводчик ещё никто не отменял. А просить кого-то отбросить свой родной язык-это вверх эгоизма и свинства. Это очень некрасиво даже с вашей стороны.

**Comment by Bezbasheniy** [comment]
> > > Несите факелы, автор согласен быть сожжонным🤣
> > 
> > 
> > Он кажется не понял, что это мы про него. Хахахах
> > (He doesn't seem to understand that we are talking about him. Hahahaha)
> 
> точно. чуваки невежественные, и видимо не только в игре.

Друг мой, ты видимо не видишь всей сути сложившейся ситуации. В таком случае я всеже удосужусь тебя просветить: Вежливость проявляеться лиш к людям, которые заслуживают хоть каплю уважения. Во всех остальных случаях это не уместно. Ты предлогаешь заблокировать человека лишь за то что он предлагает идеи, и это как по мне несусветная глупость. Куда как проще предложить свои варианты по поводу того что можно или необходимо сделать, либо обсудить предложенную другим человеком идею и тогда процес развития проекта будет протекать совсем иначе. И не будет необходимости заниматься подобной ересью, которая к слову куда более невежественная чем наши высмеевания этой темы.

**Comment by juh9870** [comment]
@Thirteen0130 
> Мне плевать. Русский-мой родной язык и я горжусь этим и я буду писать на нём. То, что вы не понимаете, это ваша проблема. Переводчик ещё никто не отменял. А просить кого-то отбросить свой родной язык-это вверх эгоизма и свинства. Это очень некрасиво даже с вашей стороны.

In case if you didn't know, people use English as a universal mean to communicate internationally. When you are asked to use English, that is to make any educated person from around the globe to be able to read your messages. Translation tools are far from perfect.

Imagine if everyone communicated and write suggestions in their native languages, now suddenly you are looking at the slew of English, German, French, Arabic, and a ton of Chinese. Please be considerate of other people and use English when communicating on the public forums.

**Comment by juh9870** [comment]
@GearGuardian91 Banning someone from suggesting is not a solution. If you oppose a suggestion, downvote it and write a comment on it explaining why you think it is a bad idea.

**Comment by Bezbasheniy** [comment]
> @Thirteen0130
> 
> > Мне плевать. Русский-мой родной язык и я горжусь этим и я буду писать на нём. То, что вы не понимаете, это ваша проблема. Переводчик ещё никто не отменял. А просить кого-то отбросить свой родной язык-это вверх эгоизма и свинства. Это очень некрасиво даже с вашей стороны.
> 
> Если вы не знали, люди используют английский язык как универсальное средство международного общения. Когда вас просят использовать английский язык, то есть сделать так, чтобы любой образованный человек со всего мира мог прочитать ваши сообщения. Инструменты перевода далеки от совершенства.
> 
> Представьте себе, если бы все общались и писали предложения на своих родных языках, теперь вы внезапно видите множество языков на английском, немецком, французском, арабском и куче китайского. Пожалуйста, будьте внимательны к другим людям и используйте английский язык при общении на общественных форумах.

Если нет возможности писать на общедоступном как в принципе в моей ситуации, то всегда есть встроенный в браузеры переводчик который позволяет прочесть текст на любом языке. В случае написания текста через переводчик, сообщение в конечном итоге выходит крайне не коректным, и ошибочным так как переводяться именно слова а не сам текст.

**Comment by vlad-ultramarine** [comment]
use Yandex translater app

**Comment by Inedilarion** [comment]
🤣

**Comment by juh9870** [comment]
Anyway, the original suggestion was not made in a good faith, and the whole discussion is also extremely counter-productive. I want to ask @PavelZinchenko to close this issue, since there is nothing productive to be said here beyond further exchange of insults.

**Comment by Thirteen0130** [comment]
> @Thirteen0130
> 
> > Мне плевать. Русский-мой родной язык и я горжусь этим и я буду писать на нём. То, что вы не понимаете, это ваша проблема. Переводчик ещё никто не отменял. А просить кого-то отбросить свой родной язык-это вверх эгоизма и свинства. Это очень некрасиво даже с вашей стороны.
> 
> На случай, если вы не знали, люди используют английский как универсальное средство международного общения. Когда вас просят использовать английский, это делается для того, чтобы любой образованный человек со всего мира мог прочитать ваши сообщения. Инструменты перевода далеки от совершенства.
> 
> Представьте, что если бы все общались и писали предложения на своих родных языках, то теперь внезапно вы смотрите на множество английских, немецких, французских, арабских и тонну китайских. Пожалуйста, будьте внимательны к другим людям и используйте английский при общении на публичных форумах.

You are right here. I agree. He was just pretty rude to everyone. And I had no desire to meet him halfway. Even for the sake of compromise

Здесь вы правы. Я согласен. Просто он был довольно груб со всеми. И у меня не было никакого желания идти ему навстречу. Даже ради компромисса

**Comment by Bezbasheniy** [comment]
А баба яга против, мисия по сожжению Жанны Дарк еще не окончена🤣🤣🤣

**Comment by Thirteen0130** [comment]
> А баба яга против, мисия по сожжению Жанны Дарк еще не окончена🤣🤣🤣

Безбашенный, ты реально безбашенный. У него и так пукан взорвался и дымит. Хватит ему на сегодня. Если что, мы его ник запомним и в следующий раз его подожжём

**Comment by juh9870** [comment]
@Bezbasheniy @Thirteen0130 GitHub is not a place for you to harass someone. Even if they have bad takes you don't agree with, please stay civil and keep your criticism constructive. The way you responded to this post made you look no better than the thread author, from the outside point of view.

**Comment by Thirteen0130** [comment]
> @Thirteen0130 @Bezbasheniy ГитХаб - не место для того, чтобы кого-то преследовать. Даже если у них есть плохие дубли, с которыми вы не согласны, пожалуйста, будьте вежливы и держите свою критику конструктивной. То, как вы отреагировали на этот пост, сделало вас не лучше автора темы со стороны.

Oh, don't push it. Let's have some fun.

 Ой, не нагнетайте. Дайте повеселиться

**Comment by Bezbasheniy** [comment]
> @Bezbasheniy @Thirteen0130GitHub — это не место, где можно кого-то преследовать. Даже если у них есть плохие мнения, с которыми вы не согласны, пожалуйста, сохраняйте вежливость и сохраняйте конструктивность критики. То, как вы ответили на этот пост, со стороны показало, что вы выглядите не лучше автора темы.

Приношу свои извинения за недопонимание, но я всеголиш с юмором отреагировал на данную тему, не более того.

**Comment by Bezbasheniy** [comment]
> > А баба яга против, миссия по сожжению окон Жанны Дарк еще не чена🤣🤣🤣
> 
> Безбашенный, ты действительно безбашенный. У него и так пукан взорвался и дымит. Хватит ему сегодня. Если что, мы его ник запомним и в следующий раз его подожжём

Понял, принял, записал🤣

**Comment by Thirteen0130** [comment]
> > @Thirteen0130 @Bezbasheniy ГитХаб - не место для того, чтобы кого-то преследовать. Даже если у них есть плохие дубли, с которыми вы не согласны, пожалуйста, будьте вежливы и держите свою критику конструктивной. То, как вы отреагировали на этот пост, сделало вас не лучше автора темы со стороны.
> 
> О, не настаивай. Давай повеселимся.
> 
> Ой, не нагнетайте. Дайте повеселиться

Ps. for some reason, the translator does not translate the words accurately. Possible inaccuracies. It was humor and sarcasm. I'm sorry if I offended anyone. I admit, we played too much and went too far.

**Comment by TaipionsSalemClient** [comment]
Yes, I really wonder why...

...also yes, making english mandatory in this place would be a very good thing,

**Comment by PavelZinchenko** [comment]
I began playing a couple of months ago without using any cheats and am currently around level 300. I personally test every new feature; I don’t just implement suggestions blindly. In my opinion, the game is quite balanced and could be released as it is. I don’t understand why people complaining so much.
Also, I’m working solo, which limits my performance. If an issue remains unresolved for a while, it doesn’t mean I’m ignoring it.

**Comment by Zen9890** [comment]
> I began playing a couple of months ago without using any cheats and am currently around level 300. I personally test every new feature; I don’t just implement suggestions blindly. In my opinion, the game is quite balanced and could be released as it is. I don’t understand why people complaining so much. Also, I’m working solo, which limits my performance. If an issue remains unresolved for a while, it doesn’t mean I’m ignoring it.

I think that main problem people complain about is that certain weaponry and ships prevail too much, for example, current best strat for arena is to Annih ray enemy base before they annih ray your base, turning gamemode that might cause some intense gameplay against much better geared enemies into space western draw simulator

**Comment by Inedilarion** [comment]
> In my opinion, the game is quite balanced and could be released as it is. I don’t understand why people complaining so much.

To see the overall picture of the state of the game and its balance, you need to try all the options it provides. And alone it is almost impossible. Communities on VK and Discord accumulate the experience of hundreds, if not thousands, of players over the years. They are also always looking for the best options for builds, weapons, ships, etc. In such conditions, all balance problems very quickly and clearly come to light, which is extremely difficult to achieve in a solo game, because comparing different options is very labor-intensive, long and approximate.

**Comment by PavelZinchenko** [comment]
> I think that main problem people complain about is that certain weaponry and ships prevail too much, for example, current best strat for arena is to Annih ray enemy base before they annih ray your base, turning gamemode that might cause some intense gameplay against much better geared enemies into space western draw simulator

Annihilation ray (500 heat+200 corrosive) isn't much stronger than Death ray (500 heat) and have the same attack range. Is it still too powerful?

**Reply by Zen9890** [reply]
It has infinite enemy penetration and cannot be countered by fodder because it doesnt aim at random shit like DR does

**Reply by Inedilarion** [reply]
It is much stronger: 1) twice less cooldown. 2) it deals full its DPS for starbase slightly longer then 2 seconds. So! It has 3x more damage then death ray and twice less cooldown. It stronger BY 6 TIMES!

**Reply by Inedilarion** [reply]
Live tests:

https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/66635560/2c1012d7-9479-4ee6-865f-88b74448901d

![Screenshot_2024-05-13-22-10-21-992_com ZipasGames Frontier](https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/66635560/df987366-0452-42f7-9eff-0384e415efa0)

Death ray damage without scientist boost is 107082/800*500 = near 67k

https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/66635560/b4046a9c-333e-43f3-93f5-cf0b2df9022b

![Screenshot_2024-05-13-22-10-04-958_com ZipasGames Frontier](https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/66635560/0478df78-0aea-4938-b22a-60e884c81e19)

So, annihilation ray deals 220k damage that it more than 3 times more than death ray

**Reply by IsmollCheeKen** [reply]
Suggestion by ChaosFred:

Increase beam length, reduce projectile speed, and Reduce damage. So it becomes a more unique weapon instead of just your standard base killer.

**Comment by Tree3324417** [comment]
this is so funny and points out a lot about the state of this game's community lol. 

if anything, i would like pavel to get back to updating event horizon. much easier place to screw around and make bugs without too much player complaints and directed hate like this :)

++to gearguardian, i never had any sort of hostile intent to you. that blocking got me entirely confused. whatever it is, calm down and give me at least an explanation before you call me off as 'laughing about your suggestion' (???)
